content/en/docs/bootstrap/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: "Bootstrap"
+linkTitle: "Bootstrap Flux"
+weight: 42
+---
```

Comment:
Do we want some text here to explain what's all done as part of the Bootstrap phase? And maybe link out to the `flux bootstap` CLI reference too, just in case?

Reply:
Yes but I think that should be done in a separate PR